src/main/java/backend/resource/TurboIssue.java

```diff
@@ -338,6 +339,28 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return Optional.empty();
     }
 
+    /**
+     * Matching is done by matching all words separated by space in query
+     * @param issues
+     * @param query
+     * @return list of issues that contains the query
+     */
+    public static List<TurboIssue> contains(List<TurboIssue> issues, String query){
+        List<String> queries = Arrays.asList(query.split("\\s"));
+        return issues.stream()
+            .filter(i -> Utility.containsIgnoreCaseMultipleWords(i.getTitle() + " " + i.getId(), queries))
```

Comment:
What I meant was changing containsIgnoreCaseMultipleWords, and making it an overload of containsIgnoreCase instead. Conceptually the operation would then just be extended to lists.